probe-rs/src/session.rs

```diff
@@ -724,15 +724,34 @@ impl Drop for Session {
         if let DebugSequence::Arm(sequence) = &self.target.debug_sequence {
             let sequence = sequence.clone();
 
-            // Note(unwrap): We already verified we're using an arm debug sequence, so we should
-            // always be able to access the arm probe interface.
-            let interface = self.get_arm_interface().unwrap();
+            if let Err(err) = { 0..self.cores.len() }.try_for_each(|i| {
+                // Note(unwrap): We already verified we're using an arm debug sequence, so we should
```

Comment:
This note seems futile now?

Reply:
It belongs to line 743.

Reply:
I would prefer to move it down in that case :)